k8s.gcr.io/ensure-prod-gclb.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates & configures the GCLB in front of the GCS bucket.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH=$(readlink -f "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
+. "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
```

Comment:
```suggestion
usage() {
```

Reply:
You're right - apparently `function` is not considered correct.  Removed.